### DISCUSSION TITLE
vitepress v1.0.0-alpha.5

### DISCUSSION THREAD

**Discussion by MarkusKeck** [vuejs/vitepress#1099] [Q&A] [answered]
Is there already a release date for v1.0.0-alpha.5?
There are many important changes which I need for my current project.
Sadly I could not find any information or date when the new release is planned.

**Comment by jevantang** [comment]
Yes, when it will be released.

I need a few features that have been merged into the repository, I wonder when the release will be?

**Comment by Charles7c** [comment]
It should be soon.

**Reply by MarkusKeck** [reply]
Thats what I thought 2 weeks ago. I cannot find a roadmap or anything. Besides the one post from Evan You which just raises more questions rather than making actual decisions, there is no information on whats planned or when.

**Reply by Charles7c** [reply]
It is never too late to have a delicious supper. I saw some action on the `chore/vite-3` branch.
![image](https://user-images.githubusercontent.com/25446948/184531905-705325c7-2bb5-4f58-9d86-14cbbb351383.png)

**Comment by Charles7c** [comment, marked as answer]
It is coming. :)
![image](https://user-images.githubusercontent.com/25446948/184790141-dff0acd8-061a-4927-9c6a-d8292cbb1808.png)
